pkg/helm/helm_test.go

```diff
@@ -59,52 +63,56 @@ var (
 	impl          = new(Impl)
 )
 
-func TestHelmCache(t *testing.T) {
+func TestHelmRegistry(t *testing.T) {
+	tmpChart := os.Getenv("TMP_CHART")
 	var err error
-	origChart := buildTestChart()
 
-	err = impl.ChartSave(origChart, chartRef)
-	assert.Nil(t, err)
-	Log(t, "save chart", err)
-
-	chart, err := impl.ChartLoad(chartRef)
-	assert.Nil(t, err)
-	Log(t, "load chart", err)
-
-	err = impl.ChartRemove(chartRef)
-	assert.Nil(t, err)
-	Log(t, "remove chart", err)
-
-	assert.Equal(t, origChart.Metadata.Name, chart.Metadata.Name, "expected loaded chart equals saved chart")
-}
-
-func TestHelmRegistry(t *testing.T) {
 	// Test should only run as integration test if registry is available
 	if _, isSet := os.LookupEnv("DOCKER_HOSTNAME"); !isSet {
 		t.Skip("No integration environment found. Skipping test...")
 	}
 
-	var err error
-	origChart := buildTestChart()
+	tmpDir, err := ioutil.TempDir("", "test-helm-")
+	if err != nil {
+		t.Errorf("Unable to create temporary directory: %s", err)
+	}
+	defer os.RemoveAll(tmpDir)
+
+	pulledChartDestPath := path.Join(tmpDir, "pulledChartDir")
+	packedChartDir := path.Join(tmpDir, "packedChartDir")
+	err = os.Mkdir(pulledChartDestPath, 0700)
+	if err != nil {
+		t.Errorf("Unable to setup test temp charts directory: %s", err)
+	}
+	err = os.Mkdir(packedChartDir, 0700)
+	if err != nil {
+		t.Errorf("Unable to setup test temp charts directory: %s", err)
+	}
 
 	if username != "" && password != "" {
+		Log(t, "user "+username+"pass "+password+"host "+hostname, err)
```

Comment:
We should probably not log passwords. Even if this is an integration test.

Reply:
Off course. probably leftover from testing. Thanks